cli/commands/project.go

```diff
@@ -0,0 +1,127 @@
+package commands
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+
+	"github.com/phase2/rig/cli/util"
+	"github.com/urfave/cli"
+)
+
+type Project struct {
+	BaseCommand
+}
+
+/*
+Here is a sample config file
+
+```
+version: 1.0
+namespace: project-name
+scripts:
+  build: "some build command && another build command"
+  start: "bin/start.sh"
+  stop:  "docker-compose stop"
+  clean: "./cleanup.sh"
+```
+*/
+
+func (cmd *Project) Commands() cli.Command {
+	flags := []cli.Flag{
+		cli.StringFlag{
+			Name:   "config",
+			Value:  "./.outrigger.yml",
+			Usage:  "Path to the project-specific configuration for Outrigger.",
+			EnvVar: "RIG_PROJECT_CONFIG_FILE",
+		},
+	}
+
+	command := cli.Command{
```

Comment:
So what happens if you just call `rig project`? I was thinking this would list out all the scripts to run.

`rig project` list all scripts
`rig project build` would run the build script
`rig project clean` would run the clean script

You are anticipating running `rig project run` to get a list of all scripts? and `rig project run build` to run the configured build script?